.github/workflows/lint.yml

```diff
@@ -71,15 +71,17 @@ jobs:
         working-directory: build
         run: |
           export CONAN_LLVM_GIT_CACHE="${{ runner.temp }}/llvm-project"
-          conan install .. -pr:h default -pr:b default
-          if [ $? -ne 0 ]
+          conan info .. -pr:h default -pr:b default --build=missing
+          build_requires=$(conan info .. -pr:h default -pr:b default --build=missing)
```

Comment:
Why are you running the same `conan info` command twice?

Reply:
Print debugging and comparing the stdout/stderr. This specific instance is a typo.